models/action.go

```diff
@@ -896,6 +896,11 @@ func mirrorSyncAction(e Engine, opType ActionType, repo *Repository, refName str
 	}); err != nil {
 		return fmt.Errorf("notifyWatchers: %v", err)
 	}
+
```

Comment:
You should move these to https://github.com/go-gitea/gitea/blob/master/models/action.go#L943 because it should be with webhook.